### ISSUE TITLE
timelapse camera fails writing to the sdcard on a ESP-CAM with MicroPython v1.13-4-g3ff707927

### ISSUE THREAD

**Issue by aguaviva** [lemariva/uPyCam#11]
timelapse camera fails writing to the sdcard on a ESP-CAM with MicroPython v1.13-4-g3ff707927-dirty on 2020-09-06; ESP32 CAM with ESP32.
 
It turns out that with this firmware you need to mount the sd card like this

```
import machine, uos
uos.mount(machine.SDCard(slot=2, width=1, sck=14, mosi=15, miso=2,  cs=13), "/sd")
```

Same thing for a TTGO T8 v1.7
See https://github.com/micropython/micropython/issues/4722#issuecomment-627641988 for more details.

**Comment by aguaviva** [comment]
Using a different SDCard all worked fine. 

I confirm that the following config works for a ESP-CAM

**Comment by lemariva** [comment]
Thanks for the tip. Solved with commit: de1351a5591c31107e9a3f5ebe80a3bc56dec14e
I've chosen `slot=3` because `slot 2` is used with the camera.
